prelexer.cpp

```diff
@@ -434,6 +434,9 @@ namespace Sass {
           alternatives <
             alnum,
             exactly <'/'>,
+            exactly <'\\'>,
```

Comment:
Do you really want to be that strict? Seems to work fine and not sure how ruby sass really does it ... this way it also accepts certain escape sequences (which I guess is correct). Feel free to create a more specific spec test ;)